src/containers/ResetPasswordContainer.jsx

```diff
@@ -0,0 +1,129 @@
+import React, { useState, useEffect } from 'react';
+import { useNavigate, useParams } from 'react-router-dom';
+
+import { useDispatch, useSelector } from 'react-redux';
+import { ToastContainer } from 'react-toastify';
+import AuthBlueSide from '../components/AuthBlueSide';
+import TLogo from '../assets/images/T_Logo.png';
+import Loading from '../components/Loading';
+import { reset, resetPassword } from '../states/features/auth/authSlice';
+
+import {
+  ErrorNotification,
+  successNotification
+} from '../components/Notification';
+
+const ResetPasswordContainer = () => {
+  const [incorrectCred, setIncorrectCred] = useState(null);
+
+  const { token } = useParams();
+  const RealToken = token.replace(/-/g, '.');
+
+  const dispatch = useDispatch();
+  const navigate = useNavigate();
+  const { isLoading, isError, isSuccess, message } = useSelector((state) => {
+    return state.auth;
+  });
+
+
+
+
+  const onSubmit = (e) => {
+    e.preventDefault();
+
+    const passwordInput = e.target.elements.password.value;
+    const confirmPasswordInput = e.target.elements.confirmPassword.value;
+
+    if (passwordInput.length < 8) {
+      setIncorrectCred('Password must be at least 8 characters long.');
+      return;
+    }
+
+    if (passwordInput !== confirmPasswordInput) {
+      setIncorrectCred('Passwords do not match.');
+      return;
+    }
```

Comment:
To ensure consistency and make the most of react-hook-form functionalities, it is recommended to avoid capturing validation errors in this manner. Instead, leverage the built-in features provided by react-hook-form to handle validation errors consistently throughout your codebase.